### ISSUE TITLE
[ Aesthetics ] Remove the Controller header's background.

### ISSUE THREAD

**Issue by JorgeR81** [chrisgoringe/cg-controller#361]
I think we can remove the header's background.

It could have some transparency, like the rest of the Controller.
The header area it's not very cluttered, and it has big icons, so it should be OK.

It would give a cleaner look, more similar to the Sidebar.

---

**mockup:**

![h011b1](https://github.com/user-attachments/assets/3f54d8e9-3679-472a-90d0-aff4cf24a4ef)

---

**original:**

![h011](https://github.com/user-attachments/assets/de1d2f68-6a97-45eb-b96b-03e77d945655)

**Comment by JorgeR81** [comment]
This is how it would look, with a single row:

---

**mockup:**

- Remove the header's background 
- Remove the divider line between rows
- Rows and icons are aligned 
- The nodes are 2 pixels lower

![zzz2](https://github.com/user-attachments/assets/ed2bffae-6404-470f-9433-922838012495)

---

**original:**

![zzz1](https://github.com/user-attachments/assets/25f9c9e6-9a5e-4919-a69c-bb4d903cd58b)

**Comment by JorgeR81** [comment]
> I like it, but at the same time it feels wrong because it seems like a block that you can grab and move up and down like a node.

Yeah, in the initial version I had a darker grey.
Maybe it works better like that.

![zz2](https://github.com/user-attachments/assets/3a918f6b-6216-497a-a14c-58848fb8ce89)

**Comment by JorgeR81** [comment]
Another alternative:

- remove the background also in the bottom row 
- use 2 divider lines 

----

This would probably match the Sidebar style better.

![zz3](https://github.com/user-attachments/assets/70324d91-512b-4a82-9c31-090ad4e192af)

**Comment by JorgeR81** [comment]
This with a bottom row 4 pixels taller.
It looks more balanced

![zz3b](https://github.com/user-attachments/assets/d8b44c2d-ee36-43f8-8b34-c4ec32d76406)

**Comment by LukeG89** [comment]
I prefer the darker grey https://github.com/chrisgoringe/cg-controller/issues/361#issuecomment-2484006830

**Comment by JorgeR81** [comment]
> I prefer the darker grey https://github.com/chrisgoringe/cg-controller/issues/361#issuecomment-2484006830

Yeah, it looks good.
Having 2 dividers looks a bit busy.

Here, instead of increasing height, I removed the bottom divider.

![zz6](https://github.com/user-attachments/assets/b0d520d2-c984-4a0c-b819-785816075c8f)

**Comment by LukeG89** [comment]
> Here, instead of increasing height, I removed the bottom divider.

Yeah, it's better with a single divider. But I still prefer the grey background, maybe because it looks better with the vertical line
